pom.xml

```diff
@@ -58,6 +58,7 @@
         <!--dependency versions -->
         <checkstyle.version>8.18</checkstyle.version>
         <gson.version>2.8.2</gson.version>
+        <slf4j.version>1.7.25</slf4j.version>
```

Comment:
1.7.26 is the latest stable version 😃

Reply:
So it should have evolved in the meantime.